Altis_Life.Altis/core/cop/fn_copLights.sqf

```diff
@@ -2,130 +2,104 @@
         _lightleft lightAttachObject [_vehicle, [-0.37, 0.0, 0.56]];
     };
-
-    case "B_MRAP_01_F":
-    {
+    case "B_MRAP_01_F": {
         _lightleft lightAttachObject [_vehicle, [-0.37, -1.9, 0.7]];
     };
-
-    case "C_SUV_01_F":
-    {
+    case "C_SUV_01_F": {
         _lightleft lightAttachObject [_vehicle, [-0.37,-1.2,0.42]];
     };
-
-    case "C_Hatchback_01_sport_F":
-    {
+    case "C_Hatchback_01_sport_F": {
         _lightleft lightAttachObject [_vehicle, [-0.35,-0.2,0.25]];
     };
-
-    case "B_Heli_Light_01_F":
-    {
+    case "B_Heli_Light_01_F": {
         _lightleft lightAttachObject [_vehicle,[-0.37, 0.0, -0.80]];
     };
-
-    case "B_Heli_Transport_01_F":
-    {
+    case "B_Heli_Transport_01_F": {         
         _lightleft lightAttachObject [_vehicle, [-0.5, 0.0, 0.81]];
-     };
+    };
 };
 
-_lightleft setLightAttenuation [0.181, 0, 1000, 130];
+_lightleft setLightAttenuation [0.181, 0, 1000, 130]; 
 _lightleft setLightIntensity 10;
 _lightleft setLightFlareSize 0.38;
 _lightleft setLightFlareMaxDistance 150;
 _lightleft setLightUseFlare true;
 
-_lightright = "#lightpoint" createVehicle getPos _vehicle;
+private _lightright = "#lightpoint" createVehicleLocal getPos _vehicle;   
 sleep 0.2;
-_lightright setLightColor _lightBlue;
-_lightright setLightBrightness 0.2;
-_lightright setLightAmbient [0.1,0.1,1];
+_lightright setLightColor _lightBlue; 
+_lightright setLightBrightness 0.2;  
+_lightright setLightAmbient [0.1,0.1,1]; 
 
-switch (typeOf _vehicle) do
-{
-    case "C_Offroad_01_F":
-    {
+switch (typeOf _vehicle) do {
+    case "C_Offroad_01_F": {
         _lightright lightAttachObject [_vehicle, [0.37, 0.0, 0.56]];
     };
-
-    case "B_MRAP_01_F":
-    {
+    case "B_MRAP_01_F": {
         _lightright lightAttachObject [_vehicle, [0.37, -1.9, 0.7]];
     };
-
-    case "C_SUV_01_F":
-    {
+    case "C_SUV_01_F": {
         _lightright lightAttachObject [_vehicle, [0.37,-1.2,0.42]];
     };
-
-    case "C_Hatchback_01_sport_F":
-    {
+    case "C_Hatchback_01_sport_F": {
         _lightright lightAttachObject [_vehicle, [0.35,-0.2,0.25]];
     };
-
-    case "B_Heli_Light_01_F":
-    {
+    case "B_Heli_Light_01_F": {
         _lightright lightAttachObject [_vehicle,[0.37, 0.0, -0.80]];
     };
-
-    case "B_Heli_Transport_01_F":
-    {
+    case "B_Heli_Transport_01_F": {         
         _lightright lightAttachObject [_vehicle, [0.5, 0.0, 0.81]];
-     };
+    };
 };
-
-_lightright setLightAttenuation [0.181, 0, 1000, 130];
+  
+_lightright setLightAttenuation [0.181, 0, 1000, 130]; 
 _lightright setLightIntensity 10;
 _lightright setLightFlareSize 0.38;
 _lightright setLightFlareMaxDistance 150;
 _lightright setLightUseFlare true;
 
-//ARE YOU ALL HAPPY?!?!?!?!?!?!?!?!?%#?@WGD?TGD?BN?ZDHBFD?GA
 _lightleft setLightDayLight true;
 _lightright setLightDayLight true;
 
-_leftRed = true;
-while{ (alive _vehicle)} do
-{
+_leftRed = true;  
+while { (alive _vehicle) } do  
```

Comment:
Make sure to keep in line with the general style (same line brace, as you've done in the other file. Can remove the parenthesis here (and whitespace).